lib/rollbax/client.ex

```diff
@@ -71,7 +72,7 @@ defmodule Rollbax.Client do
   def handle_cast({:emit, event}, %{enabled: true} = state) do
     case compose_json(state.draft, event) do
       {:ok, payload} ->
-        opts = [:async, pool: @hackney_pool]
+        opts = [:async, pool: @hackney_pool, proxy: state.proxy]
```

Comment:
Shouldn't you add the `:proxy` option only if `state.proxy` is not `nil`?

Reply:
Oh boy, my fault, I see you mentioned this in your description. I'd still rather not pass the `proxy` option if there's no `proxy`.